### PR TITLE
Another fix for resetting the total time on a new follow up ticket

### PR DESCRIPTION
**Pull request by swmichaels**
@zendesk/vegemite 

This is based off PT #1902911

The Time Tracking app in its current state is causing new follow-up tickets to inherit total handle time, which is not what we want. This appears to be caused by custom field value change events now being inherited in follow-up tickets. The the app was previously checking for the absence of these events to grab the total handle time.

I have spun up a quick fix here to bring this to your attention, so there's probably some optimization or refactoring to be done yet. Please let me know your thoughts.

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -179,20 +179,35 @@
...
+        
+        // Because of new change with how events are carried over to follow-up
+        // tickets (https://github.com/zendesk/zendesk/pull/2638) we only want
+        // the new audits
+        
+        var audits = _.filter(this.store('audits'), function(audit) {
+          return (!audit.via || 
+                  !audit.via.source || 
+                  audit.via.source.rel !== 'follow_up');
+        }, false);
```
Comment (danielbreves) [inline app.js]: If we do this do we still need the `isFollowUp` check above? Could we do `isFollowUp = !!audits.length`?

File: app.js [hunk 2]
```diff
@@ -179,20 +179,35 @@
...
       if (isFollowUp) {
-        var audits = this.store('audits'),
-            isThisEvent = function(event) {
-              return event.field_name == this.storage.totalTimeFieldId;
-            };
-        for (var i = 0; i < audits.length; i++) {
-          var audit = audits[i],
+        
+        // Because of new change with how events are carried over to follow-up
+        // tickets (https://github.com/zendesk/zendesk/pull/2638) we only want
```
Comment (danielbreves) [inline app.js]: Not sure I understand how this PR caused the issue.

Comment (swmichaels) [inline app.js]: Whoops, I missed a thing... https://github.com/zendesk/zendesk/pull/26389 is the correct one. I am out of the office for the evening, but I can correct tomorrow

File: app.js (hunk 1 above)
Comment (swmichaels) [inline app.js]: Yeah, this is doing a lot of unnecessary checks. I will implement `!!audits.length`

**Comment by swmichaels** [comment]
I've improved this a bit by separating the two types of audits. This turned out to be useful later to pass only the new audits for timelog parsing. @danielbreves, thoughts?

File: app.js [hunk 3]
```diff
@@ -172,31 +172,50 @@
...
-              return event.field_name == this.storage.totalTimeFieldId;
-            };
-        for (var i = 0; i < audits.length; i++) {
-          var audit = audits[i],
+      var status = "";
+      var timeDiff;
+
+      // separate follow up audits from new audits
+      var follow_up_audits = [];
+      var new_audits = [];
```
Comment (danielbreves) [inline app.js]: Prefer camelCase over snake case in JS.

**Pull request merged**